src/components/Charts/Dashboard.tsx

```diff
@@ -40,27 +39,44 @@ export class Dashboard<T extends LineInfo> extends React.Component<Props<T>, Sta
     };
   }
 
+  renderCharts(charts: ChartModel[]) {
+    const heightChart = (window.innerHeight - TOP_PADDING - SPACE_CARD) / 2;
```

Comment:
Instead to use this strategy, I suggest to look how the RenderComponentScroll is doing it in the overview tab and just use same way here.

Reply:
It's the same strategy @lucasponce 

https://github.com/kiali/kiali-ui/blob/master/src/components/Nav/Page/RenderComponentScroll.tsx#L31-L34